packages/react/src/components/Button/Button.mdx

```diff
@@ -67,12 +67,6 @@ Knappene kommer i to hovedfarger `first` og `second`. Hvis du har overstyrt farg
 
 <Canvas of={ButtonStories.Danger} />
 
-### Success
-
-`Success`-fargen bruker vi bare når brukeren fullfører en prosess.
-
-<Canvas of={ButtonStories.Success} />
-
```

Comment:
We decided not to remove this story yet, but add text saying we will remove it later on